src/dxvk/dxvk_util.cpp

```diff
@@ -71,6 +71,41 @@ namespace dxvk::util {
       }
     }
   }
+
+
+  void packImageData(
```

Comment:
You should make the above packImageData just call this with the info it gets.

Reply:
If anything, I'd want everything to use the path below (the one with 8 parameters), if that works. Was too lazy to deal with it yet though.

Reply:
I think having some helper that just calculates and passes it in with less args is fine. Calculating these values everywhere would kinda suck.

Reply:
I agree with Josh here.

Reply:
I'm not really sure what's going on with the d3d9 side but originally these functions were meant to just tightly pack image data so we could use it with `vkCmdCopyBufferToImage`. Looking at what this does in more detail, it's a completely different use case.

Reply:
We're kinda doing the opposite here. "unpacking" tightly packed data. Would you prefer a separate function for that?

Reply:
My problem with this whole thing is that we still have *two* functions doing more or less the thing with different capabilities. I'd rather just have one.

Reply:
So basically extend the multi-plane variant and use that instead?